Octokit/Models/Request/SearchUsersRequest.cs

```diff
@@ -165,15 +168,18 @@ public enum UserInQualifier
         /// Search by the username
         /// </summary>
         [SuppressMessage("Microsoft.Naming", "CA1702:CompoundWordsShouldBeCasedCorrectly", MessageId = "Username")]
+        [Parameter(Value = "login")]
```

Comment:
This might've been a bug as well:

> Qualifies which fields are searched. With this qualifier you can restrict the search to just the username (`login`), public email (`email`), full name (`fullname`), or any combination of these.
> \- https://developer.github.com/v3/search/#search-users